README.md

```diff
@@ -61,8 +61,14 @@ git submodule update --init --recursive
 mkdir build
 cd build
 
-cmake .. -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang
-make
+cmake .. \
+ -DCMAKE_CXX_COMPILER=clang++ \
+ -DCMAKE_C_COMPILER=clang \
+ -DCMAKE_INSTALL_PREFIX=<install location> 
+ # optional: -DCMAKE_BUILD_TYPE=<Debug(default), Release, RelWithDebInfo>
+
+make -j
```

Comment:
```suggestion
make
```
`-j` option runs "infinite" amount of jobs parallel. That could lead to troubles - e.g. the build process run out of memory.